pgdrive/scene_creator/blocks/bottleneck.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+from pgdrive.scene_creator.lane.straight_lane import StraightLane
+from pgdrive.utils.scene_utils import check_lane_on_road
+from pgdrive.pg_config.parameter_space import BlockParameterSpace, Parameter
+from pgdrive.pg_config.pg_space import PGSpace
+from pgdrive.scene_creator.blocks.block import Block, BlockSocket
+from pgdrive.scene_creator.blocks.create_block_utils import CreateAdverseRoad, CreateRoadFrom, create_bend_straight, \
+    ExtendStraightLane, create_wave_lanes
+from pgdrive.scene_creator.lane.abs_lane import LineType
+from pgdrive.scene_creator.road.road import Road
+
+
+class Bottleneck(Block):
+    """
+    This block is used to change thr lane num
+    """
+    ID = None
+    SOCKET_NUM = 1
+    PARAMETER_SPACE = PGSpace(BlockParameterSpace.BOTTLENECK_PARAMETER)
+
+    # property of bottleneck
+    BOTTLENECK_LEN = 20  # Add to parameter sapce in the future
+
+
+class InBottleneck(Bottleneck):
```

Comment:
Why don't we just call "merge"??

Reply:
![image](https://user-images.githubusercontent.com/24489294/112280315-d9768f00-8cbf-11eb-89e9-ab06df989dad.png)
I think it is more suitable... Merge is also a good class name. Both of them are ok for me